src/products/components/ProductVariantCreatePage/ProductVariantCreatePage.tsx

```diff
@@ -233,6 +229,7 @@ const ProductVariantCreatePage: React.FC<ProductVariantCreatePageProps> = ({
               <CardSpacer />
               <ProductStocks
                 data={data}
+                productVariantChannelListings={[]}
```

Comment:
Should be `productVariantChannelListings = []` in the ProductStocks component and no prop passed here

Reply:
done